### DISCUSSION TITLE
memory size of all exception messages in dotnet

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/runtime#71419] [General] [answered]
I wonder what's the total size of all the messages when throwing an exception is that comes with dotnet.
I mean
```
throw new Exception ("Message");
```

**Comment by huoyaoyuan** [comment, marked as answer]
By "all the messages" what do you mean? Bytes associated to exception instances? Or the message strings?

**Reply by Xyncgas** [reply]
The message strings

**Reply by vitek-karas** [reply]
Do you mean all of the strings which are used in exceptions across all of the libraries which ship with .NET?

Curious: What is the higher-level problem you're trying to solve?

**Reply by huoyaoyuan** [reply]
You can get a rough value by inspecting the `resx` file, or extracting the `.resources` file from decompiled CoreLib assembly.

**Reply by Xyncgas** [reply]
I see, because I am curious that is all
